src/panels/config/integrations/integration-panels/ozw/ozw-config-network.ts

```diff
@@ -0,0 +1,304 @@
+      this.hass!,
+      this.ozw_instance
+    );
+  }
+
+  private _generateServiceButton(service: string) {
+    return html`
+      <ha-call-service-button
+        .hass=${this.hass}
+        domain="ozw"
+        service="${service}"
+      >
+        ${this.hass!.localize("ui.panel.config.ozw.services." + service)}
+      </ha-call-service-button>
+    `;
+  }
+
+  protected render(): TemplateResult {
+    return html`
+      <hass-tabs-subpage
+        .hass=${this.hass}
+        .narrow=${this.narrow}
+        .route=${this.route}
+        .tabs=${ozwTabs}
+      >
+        <ha-config-section .narrow=${this.narrow} .isWide=${this.isWide}>
+          <div slot="header">
+            ${this.hass.localize("ui.panel.config.ozw.network.header")}
+          </div>
+
+          <div slot="introduction">
+            ${this.hass.localize("ui.panel.config.ozw.network.introduction")}
+          </div>
+          ${this._network
+            ? html`
+                <ha-card class="content network-status">
+                  <div class="card-content">
+                    <div class="details">
+                      ${networkOnlineStatuses.indexOf(this._network.Status) !==
+                      -1
+                        ? html`
+                            <ha-svg-icon
+                              .path=${mdiCheckCircle}
+                              class="network-status-icon online"
+                              slot="item-icon"
+                            ></ha-svg-icon>
+                          `
+                        : networkStartingStatuses.indexOf(
+                            this._network.Status
+                          ) !== -1
+                        ? html`
+                            <ha-svg-icon
+                              .path=${mdiCircle}
+                              class="network-status-icon starting"
+                              slot="item-icon"
+                            ></ha-svg-icon>
+                          `
+                        : networkOfflineStatuses.indexOf(
+                            this._network.Status
+                          ) !== -1
+                        ? html`
+                            <ha-svg-icon
+                              .path=${mdiCloseCircle}
+                              class="network-status-icon offline"
+                              slot="item-icon"
+                            ></ha-svg-icon>
+                          `
+                        : html`
+                            <ha-svg-icon
+                              .path=${mdiCircle}
+                              class="network-status-icon"
+                              slot="item-icon"
+                            ></ha-svg-icon>
+                          `}
+                      ${this.hass.localize(
+                        "ui.panel.config.ozw.common.network"
+                      )}
+                      ${networkOnlineStatuses.indexOf(this._network.Status) !==
```

Comment:
You are doing this check for the second time, store it in a var and use it for rendering. Combine it with the check for icons.